rn/release-information/release-notes-21-04-update1.adoc

```diff
@@ -45,3 +49,24 @@ The following table outlines the release particulars:
 * Fixes an issue with filesystem monitoring on pods that use NFS volumes with root squashing enabled. With this fix, NFS volumes with root squashing enabled are skipped to avoid errors.
 //#27983
 * Fixes an issue with the missing content-encoding header when WAAS is enabled.
+
+
+=== Known issues
+
+// #26088
+* Defender containers in Defender DaemonSets still display as vulnerable after upgrade.
+This happens because the auto-upgrade mechanism updates the Defender binary only, not the entire image, and the vulnerabilities exist in the image base layer.
+To resolve the issue, redeploy all Defender DaemonSets.
+
+
+=== Upcoming deprecations
+
+Starting on June 14, 2021, we'll publish https://prisma.pan.dev/api/cloud/cwpp[API documentation] for our stable endpoints only.
+The list of stable endpoints can be reviewed https://prisma.pan.dev/docs/cloud/cwpp/stable-endpoints[here].
+
+Stable endpoints have always been fully supported, and we've committed to minimizing changes to them so that your integrations don't break when upgrading to newer versions of Compute.
+
+Starting with our next release (codenamed Iverson), we'll further enhance how we support our API with versioned endpoints.
+All stable endpoints will be versioned.
+Versioned endpoints will be supported for the current release and the previous two releases (that is, Console will serve all three versions).
```

Comment:
Is this a commitment from Engineering?

Reply:
Yes.

The specific details are: Starting with Iverson, we'll support Hamilton and Iverson. And then when we we release Joule, we'll support Hamilton, Iverson, and Joule. Essentially, we won't have the full pipeline of N-2 backwards support until Joule releases. But what I've summarized in the release notes captures the essence of what we'll be offering.